src/main/java/roomescape/repository/ReservationRepositoryImpl.java

```diff
@@ -0,0 +1,82 @@
+package roomescape.repository;
+
+import roomescape.entity.Person;
+import roomescape.entity.Reservation;
+import roomescape.exception.EntityNotFoundException;
+import roomescape.service.ReservationRepository;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicLong;
+
+public class ReservationRepositoryImpl implements ReservationRepository {
+    private final Map<Long, Reservation> reservations;
+    private final AtomicLong id = new AtomicLong(1);
+
+    public ReservationRepositoryImpl() {
+        reservations = initialReservationsSetting();
+    }
+
+    private Map<Long, Reservation> initialReservationsSetting() {
+        Person person = new Person("brown");
+        Reservation reservation1 = new Reservation(
+                id.getAndIncrement(),
+                person,
+                LocalDate.of(2023,1,1),
+                LocalTime.of(10, 0,0)
+        );
+        Reservation reservation2 = new Reservation(
+                id.getAndIncrement(),
+                person,
+                LocalDate.of(2023,1,2),
+                LocalTime.of(11, 0,0)
+        );
+        Reservation reservation3 = new Reservation(
+                id.getAndIncrement(),
+                person,
+                LocalDate.of(2023,1,3),
+                LocalTime.of(12, 0,0)
+        );
+
+
+        return new HashMap<Long, Reservation>() {{
+            put(reservation1.getId(), reservation1);
+            put(reservation2.getId(), reservation2);
+            put(reservation3.getId(), reservation3);
+        }};
+    }
+
+    @Override
+    public Reservation save(Reservation reservation) {
+        Long newId = id.getAndIncrement();
+
+        reservation.setId(newId);
+        reservations.put(reservation.getId(), reservation);
```

Comment:
```suggestion
        reservations.put(newId, reservation);
```

`새로운 아이디를 가진 Reservation 객체를 저장한다` 라는 행위에 어울리는 파라미터는, `reservation.getId()` 보다 `newId` 가 더 직관적인 것 같아요.